### PR TITLE
Migratated dropdown component to tw utility classes

### PR DESCRIPTION
**Pull request by MaruaEl**
https://fxsolutions.atlassian.net/browse/ONL-4625

### CODE REVIEW

File: src/components/ec-dropdown/ec-dropdown.story.js [hunk 1]
```diff
@@ -98,10 +98,10 @@ stories.add('all', () => ({
     cta: action('CTA'),
   },
   template: `
-    <div class="ec-grid ec-p--20">
+    <div class="tw-grid tw-p-20">
```
Comment (mcibique) [inline src/components/ec-dropdown/ec-dropdown.story.js:101]: You don't need `tw-grid` on this level, there are no `tw-col-*` children in it.

File: src/components/ec-dropdown/ec-dropdown.story.js [hunk 2]
```diff
@@ -194,12 +194,12 @@ stories.add('all', () => ({
           </ec-dropdown>
         </div>
       </div>
-      <div class="ec-grid__row ec-p--12">
+      <div class="tw-grid tw-p-12">
```
Comment (mcibique) [inline src/components/ec-dropdown/ec-dropdown.story.js:197]: This element now seems broken on the story. I think it should be inside of the 
```
  <h2>Single</h2>
  <div class="tw-grid">
    ....
  </div>
```
and have class `tw-col-12` instead of `tw-grid`.

File: src/components/ec-dropdown/ec-dropdown.story.js [hunk 3]
```diff
@@ -148,26 +148,26 @@ stories.add('all', () => ({
...
-        <div class="ec-col-4">
+        <div class="tw-col-4">
           <ec-dropdown
             :items="items"
             label="Single value - with CTA"
             placeholder="Single value - with CTA"
             v-model="selected">
             <template #cta>
-              <a href="#" @click.prevent="cta()" style="display: block; padding: 8px 16px;">Do something</a>
+              <a href="#" @click.prevent="cta()" class="tw-block tw-py-8 tw-my-16">Do something</a>
```
Comment (matallo) [inline src/components/ec-dropdown/ec-dropdown.story.js:166]: why the change to margin?

**Comment by MaruaEl** [comment]
@mcibique have I forget anything from your comments? If you are happy with everything I will merge this. Thanks
